bert/yamls/glue/mcloud_run.yaml

```diff
@@ -0,0 +1,105 @@
+cluster: null  # Name of the cluster to use for this run
+gpu_type: a100_80gb # Type of GPU to use (we use a100_80gb)
+gpu_num: 8  # Number of GPUs to use
+image: mosaicml/composer:0.11.1
+integrations:
+  - integration_type: "git_repo"
+    git_repo: mosaicml/benchmarks
+command: |
+  cd benchmarks/bert
+  pip install -r requirements.txt
+  composer main.py /mnt/config/parameters.yaml
+
+# `parameters` copied from `glue/mosaic-bert-base-uncased.yaml`
+parameters:
+  # Whether to run the various GLUE jobs serially or in parallel (use parallel=True to take advantage of multiple GPUs)
+  parallel: true
+
+  # Basic run configuration, additional details will be added to this name for each GLUE task, and each random seed
+  base_run_name: *run_name # Determines how runs are saved and logged in W&B
+  default_seed: 19
+  precision: bf16
+
+  # Tokenizer for dataset creation
+  tokenizer_name: &tokenizer_name bert-base-uncased
+
+  # Base model config
+  model:
+    name: mosaic_bert
+    pretrained_model_name: *tokenizer_name
+    tokenizer_name: *tokenizer_name
+
+  # Loading
+  starting_checkpoint_load_path: null # Fill this in with the composer checkpoint from the end of pre-training a Mosaic BERT
+  local_pretrain_checkpoint_folder: ./local-bert-checkpoints/
+
+  # Saving
+  save_finetune_checkpoint_prefix: ./local-finetune-checkpoints/ # (local)
+  # save_finetune_checkpoint_prefix: s3://<bucket>/remote-finetune-checkpoints # (remote)
+  save_finetune_checkpoint_folder: ${save_finetune_checkpoint_prefix}/${base_run_name}
+
+  # Loggers
+  loggers:
+    wandb: # (Comment this block to disable W&B logging)
+      project: null # Fill this in if using W&B
+      entity: null # Fill this in if using W&B
+
+  # Callbacks
+  callbacks:
+    lr_monitor: {}
+    speed_monitor: {}
+
+  # Scheduler
+  scheduler:
+    name: linear_with_warmup
+    t_warmup: 0.06dur
+
+  # Algorithms
+  algorithms:
+    fused_layernorm: {}
+
+  # Task configuration
+  tasks:
+    mnli:
+      # Specify any extra task-specific arguments for the trainer here
+      trainer_kwargs:
+        # We keep one MNLI checkpoint locally so that we can start finetuning of 
+        # RTE, MRPC and STS-B from the MNLI checkpoint
+        save_num_checkpoints_to_keep: 1
+    rte:
+      seeds: [19, 8364, 717, 10536, 90166]
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    qqp:
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    qnli:
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    sst2:
+      seeds: [19, 8364, 717]
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    stsb:
+      seeds: [19, 8364, 717, 10536, 90166]
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    mrpc:
+      seeds: [19, 8364, 717, 10536, 90166]
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
+    cola:
+      seeds: [19, 8364, 717, 10536]
+      trainer_kwargs:
+        save_num_checkpoints_to_keep: 0
```

Comment:
You don't want to include your slightly modified GLUE parameters?

```
    mrpc:
      seeds: [19, 8364, 717, 10536, 90166]
      trainer_kwargs:
        save_num_checkpoints_to_keep: 0
    cola:
      seeds: [19, 8364, 717, 10536]
      trainer_kwargs:
        save_num_checkpoints_to_keep: 0
        max_duration: 40ep
        batch_size: 128
   ```

Reply:
(these are the updated details/hyperparameters I have been using in my GLUE sweeps)

Reply:
It is looking less likely that we need the specific HPs, so we can leave it out for now. Also, we will update the recipes once the results are finalized.